config/s2i/jenkins/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM openshift/jenkins-2-centos7
```

Comment:
Is this file necessary? Doesn't plugins get installed regardless?

Reply:
They do, but if a project wants to use this image and has other plugins to install, plugins.txt will be overridden.

Reply:
@14rcole same question

Reply:
They do, but if a project wants to use this image and has additional plugins to install this allows plugins.txt to be overridden

Reply:
I'll have to go back and look again, but at the time I created this file, I think I understood the requirement to be a minimal base that functions as a starting point for linchpin, ci-pipeline, etc. master images.  Is this more of a hybrid of everything the jenkins masters could possibly need?

Reply:
Jenkins Core needs to be updated and with the set of plugins that we use is fine but it still needs updating regardless.
Does that make sense?

Reply:
Yeah